### DISCUSSION TITLE
Advertise honor to gold feature in a regular server announcement.

### DISCUSSION THREAD

**Discussion by minthes** [chromiecraft/chromiecraft#7629] [Suggestions]
Players need more incentives to participate in Battlegrounds, but it’s important to recognize that some effective incentives already exist—they just aren’t widely known. For instance, a PvP-focused friend of mine, who joined the server months ago, only yesterday discovered the honor-to-gold feature.

The solution is simple: regularly announce these existing incentives a few times throughout the day. By raising awareness, players will be more likely to engage in BGs, unlocking rewards they may not have realized were available. This small adjustment could make a significant impact on participation and overall PvP activity.

**Comment by Apathyxia** [comment]
200g for several hours of BGs during CtA is more or less nothing when open-world farming is easily 200-300g/hour but also 500-600g/hour as certain specs. I think even if it's announced, it doesn't really boost the activity. That is not really an incentive in my eyes. It needs to be higher to make it worthwhile. 

But then again, it doesn't hurt anyone to have it announced. I also don't want to see lots of ungeared people now doing BGs for gold. That will not make the game fun especially during off-peak hours where even one person being in blue or full pve gear can be the direct cause of loss to that team. Hence, making S1 gear for gold would at least help that, and more has been suggested [here](https://github.com/orgs/chromiecraft/discussions/7622) and many times on Discord.
